docs/quick-start/adapters.md

```diff
@@ -1,40 +1,58 @@
 <!--[meta]
 section: quick-start
-title: Configuring Adapters
+title: Database Setup and Adapters
 [meta]-->
 
-# Adapter Configuration
+## Choosing an Adapter
 
-## Choosing an adapter
+KeystoneJS currently provides two adapters for connecting to either a MongoDB or PostgreSQL database. Choose the [Mongoose Adapter](/keystone-alpha/adapter-mongoose/) for MongoDB or the [Knex Adapter](/keystone-alpha/adapter-knex/) for PostgreSQL.
 
-KeystoneJS currently provides two database adapters. Choose the [Mongoose Adapter](/keystone-alpha/adapter-mongoose/) for MongoDB or the [Knex Adapter](/keystone-alpha/adapter-knex/) for PostgreSQL.
+If you're following the [quick start guide](/quick-start/adapters), simply select the appropriate adapter for your database of choice when prompted. More information on adapter configuration can be found under the *Setup* sections.
 
 _Note_: PostgreSQL requires an additional step to create a database.
 
 ## Installing MongoDB
 
-The simplest way to install [MongoDB](https://www.mongodb.com/) is using [Homebrew](https://brew.sh/).
+### MacOS
 
-### OSX
+The simplest way to install [MongoDB](https://www.mongodb.com/) is using [Homebrew](https://brew.sh/).
 
 ```sh
 brew install mongodb
 brew services start mongodb
 ```
 
+### Windows
+
+Note this is not the only way to install MongoDB on Windows, but it is convenient for portability.
+
+1. Download the [MongoDB community edition](https://www.mongodb.com/download-center/community) **ZIP** package.
+2. Unzip the folder to a convenient location, such as `Documents\mongodb`.
+3. Create a new folder called `data` inside the `mongodb` folder. This will be where your database actually writes its data.
+
+```sh
+cd mongodb
+mkdir data
+```
+
+4. Run the following command to start the database. You can save it as a script (`.sh`) file too for convenience.
+```sh
+.\bin\mongod.exe --dbpath=C:\Users\USERNAME\Documents\mongodb\data
+```
+
```

Comment:
These instruction are really specific. I'm not sure that we want to instruct everyone to setup a portable version in this way and to run a separate script to start the DB... it should be running as a service. But more importantly I'd like to leave that to the user... Unless you can clarify why your would people to do this over the standard installation?

Reply:
I'd love to see this as an external blog post (maybe on your person site, @Vultraz?), then we could link to it from here 👍

Reply:
Ah, I don't have an external blog unfortunately. 😬 I'll drop this bit.

@MadeByMike I don't have a specific reason. It's a setup I know (I was using it for with with v4) and since there was no specific instructions for Windows, I figured I'd mention something.